test/e2e/framework.go

```diff
@@ -1296,20 +1309,65 @@ func (data *TestData) gracefulExitAntreaAgent(covDir string, nodeName string) er
 		err = data.copyPodFiles(podName, "antrea-agent", antreaNamespace, antreaAgentCovFile, covDir)
 
 		if err != nil {
-			return fmt.Errorf("error when graceful exit Antrea agent: copy pod files out, error:%v", err)
+			return fmt.Errorf("error when graceful exit Antrea agent: copy antrea-agent coverage files out, error:%v", err)
 		}
+
 	}
 	return nil
 }
 
-// gracefulExitAntreaAgent copies the Antrea agent binary coverage data file out before terminating the Pod
-func (data *TestData) copyPodFiles(podName string, containerName string, nsName string, fileName string, covDir string) error {
-	fmt.Printf("Copying file %s from pod %s podName to '%s'", fileName, podName, covDir)
+//collectAntctlCovFiles collects coverage files for the antctl binary from the Pod and saves them to the coverage directory
+func (data *TestData) collectAntctlCovFiles(podName string, containerName string, nsName string, covDir string) error {
+	// copy antctl coverage files from Pod to the coverage directory
+	cmds := []string{"bash", "-c", "find . -maxdepth 1 -name 'antctl*.out' -exec basename {} ';'"}
+	stdout, stderr, err := data.runCommandFromPod(nsName, podName, containerName, cmds)
+	if err != nil {
+		return fmt.Errorf("error when finding the coverage files for antctl: <%v>, err: <%v>", stderr, err)
+	}
+	stdout = strings.TrimSpace(stdout)
+	files := strings.Split(stdout, "\n")
+	for _, file := range files {
+		if len(file) == 0 {
+			continue
+		}
+		err := data.copyPodFiles(podName, containerName, nsName, file, covDir)
+		if err != nil {
+			return fmt.Errorf("error when copying coverage files for antctl from Pod '%s' to coverage directory '%s': error: <%v>", podName, covDir, err)
+		}
+	}
+	return nil
+}
+
+//collectAntctlCovFilesFromMasterNode collects coverage files for the antctl binary from the master node and saves them to the coverage directory
+func (data *TestData) collectAntctlCovFilesFromMasterNode(covDir string) error {
+	// copy antctl coverage files from node to the coverage directory
+	cmd := "/bin/sh -c find . -maxdepth 1 -name 'antctl*.out' -exec basename {} ';'"
```

Comment:
do we have to assume a shell for the Node? I would think calling `find` directly is just a matter of finding the right syntax

Reply:
Looks like there is no right syntax, or it's very hard to find. So feel free to ignore this comment.
I think we could just call `find . -maxdepth 1 -name 'antctl*.out'` and use the `Base()` function in the Go `path` package, but we can stick with `sh` and see if it creates an issue in the future (I doubt it will).

Reply:
I tried to use RunDockerExecCommand() function to just call find . -maxdepth 1 -name 'antctl*.out', then the output is empty even though there are 'antctl*.out' files on the master node. 

root@sc2-hs3-nsbu-ncp-dhcp--27-47:~# docker exec -t kind-control-plane find . -maxdepth 1 -name 'test'
./test
root@sc2-hs3-nsbu-ncp-dhcp--27-47:~# vi test.go
root@sc2-hs3-nsbu-ncp-dhcp--27-47:~# go run test.go
/usr/bin/docker exec -t kind-control-plane find . -maxdepth 1 -name 'test'


Then at this time, I think maybe we can stick with sh and resolve if there will be any issues in the future.